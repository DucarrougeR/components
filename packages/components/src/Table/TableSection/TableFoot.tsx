import styled from 'styled-components'
import {
  tableSectionCSS,
  tableSectionDefaults,
  TableSectionProps,
} from './tableSection'

export const TableFoot = styled.tbody<TableSectionProps>`
  ${tableSectionCSS}
`

TableFoot.defaultProps = tableSectionDefaults
