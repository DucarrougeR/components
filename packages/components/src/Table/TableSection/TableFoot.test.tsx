import 'jest-styled-components'
import React from 'react'
import { createWithTheme } from 'looker-components-test-utils'
import { TableFoot } from './TableFoot'

test('A <TableFoot> should render', () => {
  const component = createWithTheme(<TableFoot />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
