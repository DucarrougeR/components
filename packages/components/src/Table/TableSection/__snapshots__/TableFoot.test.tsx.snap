// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A <TableFoot> should render 1`] = `
.c0 {
  border: 0;
  box-sizing: border-box;
  font: inherit;
  font-size: 100%;
  margin: 0;
  padding: 0;
  vertical-align: baseline;
  text-align: left;
}

<tbody
  className="c0"
/>
`;
